internal/mage/sdk/elixir.go

```diff
@@ -157,9 +158,10 @@ func (Elixir) Publish(ctx context.Context, tag string) error {
 	var (
 		version = strings.TrimPrefix(tag, "sdk/elixir/v")
 		mixFile = "sdk/elixir/mix.exs"
-		dryRun  = os.Getenv("HEX_DRY_RUN")
 	)
 
+	dryRun, _ := strconv.ParseBool(os.Getenv("DRY_RUN"))
```

Comment:
How do you feel about passing this as an argument instead?

Reply:
Ideally, I'd really like this as a flag.

But this doesn't look like something we do at the moment :cry: https://github.com/magefile/mage/issues/397

I don't really want it as a positional arg as a string, since otherwise it looks like: `./hack/make sdk:elixir:publish v0.9.8 false`.

Reply:
Not a problem, we'll fix it soon when we modularize our CI 🙂